src/qudi/core/modulemanager.py

```diff
@@ -246,6 +248,74 @@ def _qudi_main_ref_dead_callback(self):
                      'ModuleManager.')
         self.clear()
 
+    def dump_status_variables(self):
+        """
+        Method that dumps the status variables of all active modules.
+        """
+        logger.debug(f"Dumping status variables")
+        with self._lock:
+            for _, module in self.modules.items():
+                if module.is_active:
+                    module.instance._dump_status_variables()
```

Comment:
Evil access on private method. On the first glance, we might elevate dump_status_variables() to a public function.

Reply:
The method has been elevated to a public function.